py/torch_tensorrt/dynamo/utils.py

```diff
@@ -169,7 +174,7 @@ def parse_dynamo_kwargs(kwargs: Any) -> CompilationSettings:
     return settings
 
 
-def req_torch_version(min_torch_version: str = "2.dev") -> Callable[..., Any]:
+def req_torch_version(func: Callable[P, T]) -> Callable[P, T]:
```

Comment:
Why was this changed?

Reply:
The arguments dont seem right

Reply:
reverted back. Seems like mypy passes now with no changes. Although I'm not sure why it was giving issues before